class ExceptionHandler:
    def handle(e: Exception):
        print(e)
